client/src/App.tsx

```diff
@@ -31,13 +26,8 @@ function App(): JSX.Element {
                 <Route exact path="/landingpage" component={LandingPage} />
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
-                <Route exact path="/editprofile" component={EditProfile} />
-                <Route exact path="/payment" component={Payment} />
-                <Route exact path="/availability" component={Availability} />
-                <Route exact path="/security" component={Security} />
-                <Route exact path="/settings" component={Settings} />
-                <Route exact path="/profile/pics" component={ProfilePhoto} />
                 <Route exact path="/profile" component={SitterProfile} />
+                <ProtectedRoutes path="/manageprofile" component={ManageProfile} />
```

Comment:
Please use hyphenated case for route names 
Eg: `manage-profile`

Reply:
Didn't realize that was the convention, thank you!